torch/csrc/jit/codegen/cuda/test/test_gpu_tensorcore.cpp

```diff
@@ -3242,6 +3242,184 @@ TEST_F(NVFuserTest, FusionAmpereStridedBatchedMatmulTN_CUDA) {
   TORCH_CHECK(cg_outputs[0].allclose(ref, 0.0001, 0.0001));
 }
 
+// Test an end-to-end matmul case with swizzled smem
+// data layout.
+TEST_F(NVFuserTest, FusioAmpereMatmulTNSwizzled_CUDA) {
+  NVFUSER_TEST_CUDA_ARCH_GUARD(8, 0);
+
+  Fusion fusion;
+  FusionGuard fg(&fusion);
+
+  int M = 257, N = 511, K = 136;
+
+  MatMulTileOptions gemm_tile;
+  gemm_tile.cta_tile = GemmTile(128, 128, 32);
+  gemm_tile.warp_tile = GemmTile(64, 64, 32);
+  gemm_tile.instruction_tile = GemmTile(16, 8, 16);
+
+  // [M,K]
+  auto tv0 = makeContigTensor(2, DataType::Half);
+  // [N,K]
+  auto tv1 = makeContigTensor(2, DataType::Half);
+  fusion.addInput(tv0);
+  fusion.addInput(tv1);
+
+  // [M,N,K]
+  auto tv0b = broadcast(tv0, {false, true, false});
+  auto tv1b = broadcast(tv1, {true, false, false});
+
+  auto mma_builder =
+      MmaBuilder(MmaOptions::MacroType::Turing_16_8_16, gemm_tile)
+          .layout(MmaOptions::MmaInputLayout::TN);
+
+  // Leaving both sets of mma inputs for volta outside
```

Comment:
Was this comment copy and pasted? Uncertain what you're trying to say here.

Reply:
Yes it's no longer the case in the current mma interface. Removed. Thanks.